src/plugins/intel_gna/tests/functional/shared_tests_instances/single_layer_tests/conv_low_precision.cpp

```diff
@@ -148,9 +162,9 @@ const vector<map<string, string>> configs_3_5 = {
     {{"GNA_DEVICE_MODE", "GNA_SW_EXACT"}, {"GNA_PRECISION", "I16"}, {"GNA_EXEC_TARGET", "GNA_TARGET_3_5"}},
 };
 
-const vector<map<string, string>> configs_2_0 = {
-    {{"GNA_DEVICE_MODE", "GNA_SW_EXACT"}, {"GNA_PRECISION", "I8"}, {"GNA_EXEC_TARGET", "GNA_TARGET_2_0"}},
-    {{"GNA_DEVICE_MODE", "GNA_SW_EXACT"}, {"GNA_PRECISION", "I16"}, {"GNA_EXEC_TARGET", "GNA_TARGET_2_0"}},
+const vector<map<string, string>> configs_3_6 = {
```

Comment:
I don't think we should add a new configuration set at this point. It won't be triggered anyway and will bring no new tests when it's finally triggered.

Reply:
removig this test